e2etests/tests/session.go

```diff
@@ -123,7 +123,7 @@ var _ = ginkgo.Describe("Advertisement", func() {
 			}
 		})
 
-		ginkgo.DescribeTable("Establishes sessions with cleartext password", func(family ipfamily.Family) {
+		ginkgo.DescribeTable("Establishes sessions with cleartext password and graceful restart", func(family ipfamily.Family) {
```

Comment:
can we have a separate test instead?
Also, is there benefit in testing e2e (also, looking at the other side when frr-k8s receive routes) as we did in metallb?

Reply:
> can we have a separate test instead?
I will do that sure

> is there benefit in testing e2e (also, looking at the other side when frr-k8s receive routes) as we did in metallb?
I would like first to connect metallb/frr-k8s and make the the we did there to run also in the FRR-k8s mode.
Mixed feeling if there is benefit but no objection to add. Do you have any test I can replicate or should I build from scratch?